Source/WebKit/WebProcess/WebCoreSupport/WebChromeClient.cpp

```diff
@@ -1695,9 +1695,9 @@ void WebChromeClient::requestTextRecognition(Element& element, TextRecognitionOp
 
 #endif
 
-URL WebChromeClient::applyLinkDecorationFiltering(const URL& url, LinkDecorationFilteringTrigger trigger) const
+URL WebChromeClient::applyLinkDecorationFiltering(const URL& url, LinkDecorationFilteringTrigger trigger, bool* appliedFilter) const
```

Comment:
Nit - instead of using outparams, we _usually_ prefer making the return type either a `struct`, or `std::pair`.